.github/workflows/linter.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will run linter in order to keep a consistent coding style.
+
+name: Linter
+
+on:
+    push:
+        paths: [src, "*.json"]
```

Comment:
why for JSON ?

Reply:
anyway faut qu'il se trigger quand on change les règles d'eslint et de prettier